lib/src/base-styles/_nx-lists.scss

```diff
@@ -180,10 +181,9 @@
     > .nx-chevron {
       align-self: center;
       color: #2c2c2c;
-      -ms-grid-row: 1;
+      -ms-grid-row: 1 / 2;
       -ms-grid-column: 2;
-      grid-area: actions;
-
+      grid-row: actions / blank;
```

Comment:
Is `blank` still a good name for that area?

Reply:
> Is `blank` still a good name for that area?

The way that it's working now we really don't need to specify that grid cell as unique, we can just call it `actions` so the second column is a single area: https://github.com/sonatype/sonatype-react-shared-components/pull/157/commits/07382b2985964bed8a748eaecabd24565e7ddb75